tests/e2e/specs/LicenseCode.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+
+    'Check if the p-tag with the class license-text is present': function(browser) {
+        browser
+            .init()
+            .assert.elementPresent('.control-label > span')
+            .click('.control-label > span')
+            .assert.elementPresent('.pagination-next')
+            .click('.pagination-next')
+            .assert.elementPresent('.select > select')
+            .click('.select > select')
```

Comment:
This line causes an error running tests:

> Error while running .clickElement() protocol action: element click intercepted: Element <select>...</select> is not clickable at point (871, 102). Other element would receive the click
: <span class="select">...</span>

If you click on span.select, you will get the result you were expecting, and the tests run fine.

The same happens in LicenseDetailsCard

Reply:
This test passed when I was sending a pull request, thank you for pointing this out I'll surely edit this :)

Reply:
> This test passed when I was sending a pull request, thank you for pointing this out I'll surely edit this :)

I think it passed for me before, too. Anyways, I opened an issue, and I think it's best if I assign it to you. Thank you for quick reply!